src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsInterstitialObject.java

```diff
@@ -16,6 +16,21 @@
  */
 public class AdsInterstitialObject extends AdsBaseObject {
 
+  /**
+   * Checks if the actual size (width or height) is within the "range of correctness" ;)
+   *
+   * We don't want to be pixel perfect in terms of checking the size of element because
+   * it depends on many things such as in example components of a browser's window.
+   *
+   * We also don't want to just check if the content of scalable interstitial is bigger
+   * than the original image. So, we came up with a reasonable size difference margin.
+   *
+   * The size defined here is doubled at the end.
+   *
+   * @see AdsInterstitialObject::isSizeCorrect()
+   */
+  private final int SIZE_DIFFERENCE_TOLERANCE = 1;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "SIZE_DIFFERENCE_TOLERANCE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)